### PR TITLE
HOCS 2492 Create QAX namespace

### PR DESCRIPTION
**Pull request by Paulcr-ho**
Deploy script amended for deployment to QAX.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -79,6 +82,9 @@ elif [[ "${KUBE_NAMESPACE}" == "cs-demo" ]] ; then
 elif [[ "${KUBE_NAMESPACE}" == "wcs-demo" ]] ; then
     export DNS_PREFIX=demo.wcs-notprod
     export KC_REALM=https://sso-dev.notprod.homeoffice.gov.uk/auth/realms/hocs-notprod
+elif [[ "${KUBE_NAMESPACE}" == "hocs-qax" ]] ; then
+    export DNS_PREFIX=qax.internal.cs-notprod.homeoffice.gov.uk
```
Comment (LiamMacP) [inline deploy.sh:86]: Is that what the DNS_PREFIX should be? looking at the others it looks like it should be `qax.cs-notprod`?

Comment (Paulcr-ho) [inline deploy.sh:86]:
```suggestion
    export DNS_PREFIX=qax.internal.cs-notprod
```

**Review by Paulcr-ho — COMMENTED** [review]
Removed the homeoffice.gov.uk.
We are using the internal ingress (Kube vpn).

File: deploy.sh (hunk 1 above)
Comment (Paulcr-ho) [reply deploy.sh]: Have updated the code.

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**